pkg/report/linux.go

```diff
@@ -28,8 +29,11 @@ type linux struct {
 	questionableFrame     *regexp.Regexp
 	guiltyFileIgnores     []*regexp.Regexp
 	reportStartIgnores    []*regexp.Regexp
+	multiReadIgnores      []*regexp.Regexp
 	infoMessagesWithStack [][]byte
 	eoi                   []byte
+	multiReadMutex        sync.Mutex
```

Comment:
I think we'd better move this mutex closer to multiReadIgnores.
Please also remove the empty line after it.